recipes/omr1/configs/full-tir-stage-0.yaml

```diff
@@ -0,0 +1,57 @@
+cluster: oci
+
+output_dir: /workspace/omr1
+expname: omr1
+
+problem_sdg:
+  # needs to be uploaded on cluster
+  input_file: /workspace/omr1/raw_aops_data.jsonl
+
+  generation:
+    model: /trt_models/qwen2.5-32b-instruct
+    extra_args: "++prompt_template=qwen-instruct"
+    server_type: trtllm
+    server_gpus: 8
+    server_nodes: 1
+    num_chunks: 10  # since data is big, we are parallelizing it 10x
+
+solution_sdg:
+  # this is the output of problem_generation.py
+  input_file: "{output_dir}/all-problems.jsonl"
+  suffix: tir-stage-0  # just used to put files in a separate folder
+
+  generation:
+    # you can take and convert the model from https://huggingface.co/GAIR/LIMO
+    # in our experiments we re-trained qwen2.5-32b-instruct on LIMO dataset and used the resulting model,
+    # which is similar to LIMO Qwen
+    model: /trt_models/LIMO 
+    extra_args: >
+      ++prompt_config=generic/math-tir-detailed
```

Comment:
not pushed? Also probably best to keep it inside prompts folder in recipes as it's not generally applicable